botathon/index.php

```diff
@@ -582,91 +582,104 @@ class="rd-parallax-layer section-top-75 section-md-top-150 section-lg-top-260">
 footer(false);
 ?>
 <script>
-	// var currentNav = "#nav-info"
-	// var currentAnchor = '#info'
-	var anchors = ['info','register','schedule','parts-list','teams','sponsors','contacts']
-	var curIndex = 0
-	var lastScrollTop = $(window).scrollTop()
+	let anchorSelectors = [$("#info"), $("#register"), $("#schedule"), $("#parts-list"), $("#teams"), $("#sponsors"), $("#contacts"),]
+	let navSelectors = [document.querySelector("#nav-info"), document.querySelector("#nav-register"), document.querySelector("#nav-schedule"), document.querySelector("#nav-parts-list"), document.querySelector("#nav-teams"), document.querySelector("#nav-sponsors"), document.querySelector("#nav-contacts"),]
+	let footerSelector = $('footer')
+	let curIndex = 0
+	let windowSelector = $(window)
```

Comment:
I am so used to just seeing `$(x)` everywhere that assigning these to `..selector` variables makes the code much less readable, can we just use the `$(x)` references in each place?

Reply:
Will do. I was trying to reduce the amount of jQuery when I could since querySelector is more performant (or so I'm told)

Reply:
Or did you mean replacing references to the vars with the query itself (instead of storing the jquery result in a variable)?

Reply:
More performant is less of an issue on client side code because this runs on a main thread in the user's browser and the tiny difference between a single selection here in inconsequential.

I was more commenting on the fact that `footerSelector` is a confusing variable, I'd either use `$('footer')` inline in the code or name it `$footer` because it's not clear to me what you meant by `selector` since that it's a reference to a _selector_ (i.e. function/supplier) it's a reference to an element

Reply:
Changed in 6909ab5